### PR TITLE
Fix examples not displaying figures

### PR DESCRIPTION
**Pull request by cgohlke**
## For reviewers

(Don't remove the checklist below.)

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

**Comment by cgohlke** [comment]
There are more examples not displaying the figures. Will add to this PR.

File: doc/examples/xx_applications/plot_morphology.py [hunk 1]
```diff
@@ -240,3 +240,5 @@ def plot_comparison(original, filtered, filter_name):
 # /ImageProcessing-html/topic4.htm>`_
 #
 # 3. http://en.wikipedia.org/wiki/Mathematical_morphology
+
+plt.show()
```
Comment (soupault) [inline doc/examples/xx_applications/plot_morphology.py]: I think this should belong to `plot_comparison` function.

Comment (cgohlke) [reply doc/examples/xx_applications/plot_morphology.py]: The options are to show the figures successively or at once at the end of the script. I prefer the latter but it's the choice of the maintainers...

**Comment by stefanv** [comment]
Thanks, Christoph.

**Pull request merged**